### PR TITLE
add a module to spend cpu time

### PR DESCRIPTION
**Pull request by davidlange6**

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

File: CommonTools/Utils/plugins/CPUSpender.h [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#ifndef CommonTools_Utils_CPUSpender_h
```
Comment (slava77) [inline CommonTools/Utils/plugins/CPUSpender.h]: why not everything in the .cc file?

File: CommonTools/Utils/plugins/CPUSpender.h [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+#define CommonTools_Utils_CPUSpender_h
+
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+
+class CPUSpender: public edm::EDAnalyzer {
```
Comment (slava77) [inline CommonTools/Utils/plugins/CPUSpender.h]: legacy module?

**Comment by davidlange6** [comment]
please test

**Pull request merged**